### ISSUE TITLE
Too much logging messages

### ISSUE THREAD

**Issue by PaulBoon** [IQSS/dataverse#5749]
Just installed our fork of 4.10.1 and was hoping to get a bit cleaner logging, 
however I still got a lot of warnings swamping my log file. 
 
The most annoying one I get more then once every second: 
 

> [2019-04-10T02:46:23.158+0200] [glassfish 4.1] [WARNING] [] [] [tid: _ThreadID=53 _ThreadName=jk-connector(3)] [timeMillis: 1554857183158] [levelValue: 900] [[
>   Response has already been committed, and further write operations are not permitted. This may result in an IllegalStateException being triggered by the underlying application. To avoid this situation, consider adding a Rule `.when(Direction.isInbound().and(Response.isCommitted())).perform(Lifecycle.abort())`, or figure out where the response is being incorrectly committed and correct the bug in the offending code.]]
> 

 
This has been mentioned in:   
https://github.com/IQSS/dataverse/issues/4427#issuecomment-359598863

The other one is even 'SEVERE':

> [2019-04-10T11:42:31.981+0200] [glassfish 4.1] [SEVERE] [] [javax.enterprise.resource.webcontainer.jsf.context] [tid: _ThreadID=54 _ThreadName=jk-connector(4)] [timeMilli
> s: 1554889351981] [levelValue: 1000] [[
>   javax.faces.application.ViewExpiredException: viewId:/dataverse.xhtml - View /dataverse.xhtml could not be restored.

Not sure which action causes this, but I see it quite often. 

Possible it is related to: 
"as a system administrator, I'd like glassfish to not throw exceptions from (from dataverseuser.xhtml) during normal operations" #5446

I would love to get rid of those, so the logs are better to use. 
Maybe there is a fast way to suppress them while the real problem is being fixed?

**Comment by djbrooke** [comment]
- these happen enough that they are easy to replicate
- we should make the decision about whether or not we are OK with suppressing them (performance considerations). If we try to fix it we may learn more about the cause

**Comment by mheppler** [comment]
Another fairly common warning I am seeing in every one of my local server logs:

```
[2019-07-01T11:43:46.675-0400] [glassfish 4.1] [WARNING] [] [org.glassfish.jersey.internal.Errors] [tid: _ThreadID=42 _ThreadName=admin-listener(1)] [timeMillis: 1561995826675] [levelValue: 900] [[
  The following warnings have been detected: WARNING: The (sub)resource method oaiSets in edu.harvard.iq.dataverse.api.HarvestingServer contains empty path annotation.
WARNING: The (sub)resource method harvestingClients in edu.harvard.iq.dataverse.api.HarvestingClients contains empty path annotation.
WARNING: A HTTP GET method, public javax.ws.rs.core.Response edu.harvard.iq.dataverse.api.Files.getFileMetadata(java.lang.String,java.lang.String,javax.ws.rs.core.UriInfo,javax.ws.rs.core.HttpHeaders,javax.servlet.http.HttpServletResponse,java.lang.Boolean) throws edu.harvard.iq.dataverse.api.AbstractApiBean$WrappedResponse,java.lang.Exception, should not consume any entity.
WARNING: A HTTP GET method, public javax.ws.rs.core.Response edu.harvard.iq.dataverse.api.Files.getFileMetadataDraft(java.lang.String,java.lang.String,javax.ws.rs.core.UriInfo,javax.ws.rs.core.HttpHeaders,javax.servlet.http.HttpServletResponse,java.lang.Boolean) throws edu.harvard.iq.dataverse.api.AbstractApiBean$WrappedResponse,java.lang.Exception, should not consume any entity.
WARNING: A HTTP GET method, public javax.ws.rs.core.Response edu.harvard.iq.dataverse.api.Prov.getProvFreeForm(java.lang.String,java.lang.String), should not consume any entity.
WARNING: A HTTP GET method, public javax.ws.rs.core.Response edu.harvard.iq.dataverse.api.Prov.getProvJson(java.lang.String,java.lang.String), should not consume any entity.
]]
```

That warning was included in a snippet in #3801, as well as a dozen server logs included in RT and community group emails.

**Comment by sekmiller** [comment]
I'm getting the "response has already been committed" jsf error only when I publish both the dataset and its parent dataverse at the same time ("do you want to publish both? popup), possibly because both success messages aren't displayed.  I would venture that is a fairly rare occurrence. Does anyone know of a state that would cause the error to display every second as described by Paul above?
